packages/block-editor/src/components/list-view/use-list-view-drop-zone.js

```diff
@@ -378,7 +385,12 @@ export default function useListViewDropZone() {
 	const draggedBlockClientIds = getDraggedBlockClientIds();
 	const throttled = useThrottle(
 		useCallback(
-			( event, currentTarget ) => {
+			( event, currentTarget, action ) => {
+				if ( action === 'clear' ) {
+					setTarget( null );
```

Comment:
This change ensures that when a user drags outside the drop zone area, the drop indicator line disappears. This resolves a point of confusion where a line was visible, a user releases the mouse button, and is then confused as to why nothing happened.

Reply:
Why don't we just call `setTarget(null)` directly in the `onDragLeave` event? That is, don't combine it in this throttled event.

Reply:
I tried that originally, but it wound up behaving inconsistently — sometimes it would switch it to `null` but then it seemed that the throttled event from dragging over might fire slightly after, and then reset the `setTarget`. I'm not 100% sure what order things were happening in, but moving to the throttled event resolved it.

Happy to re-test it, though, in case I missed something 🙂

Reply:
Yep, looks like if we have `setTarget( null )` directly in `onDragLeave` then the throttled event fires directly after, restoring the target. It's pretty subtle, but you should hopefully be able to see the flicker off when dragging out of the list view in this gif, before it gets displayed again:

![2023-05-19 14 49 15](https://github.com/WordPress/gutenberg/assets/14988353/81f6f286-3907-4672-a4b5-9950ec635c23)

Reply:
I think that's because we're not canceling the throttled event? Try calling `throttled.cancel()` before setting the target and see if it fixes the issue? Nvm if it's not the case 😆 .

Reply:
Ah, of course! That's working nicely, and I see now that's how the `onDragEnd` callback does it, too. Thanks @kevin940726, much appreciated! 🙇

Reply:
Nice! Thanks for double-checking! 🙇